index.js

```diff
@@ -1,11 +1,15 @@
 module.exports = {
-  extends: [
-    "eslint:recommended",
-    "plugin:@typescript-eslint/recommended"
-  ],
+  "plugins": ["jest"],
+  "extends": ["plugin:@typescript-eslint/recommended", "plugin:jest/recommended"],
```

Comment:
Why was `eslint:recommended` removed?

Reply:
Oops, this might have been a mistake when I added jest.